src/App.tsx

```diff
@@ -85,6 +87,10 @@ function App() {
                     <Route path="/testbb" element={<TestBB />} />
                     <Route path="/testss" element={<TestSS />} />
                     <Route path="/timesheet" element={<Timesheet />} />
+                    <Route
+                        path="/timesheetDetails"
```

Comment:
For now we can leave it like this but in the future it will be /timesheet/selected-date for example /timesheet/24-12-2021